src/Embed/Maps.php

```diff
@@ -682,8 +682,16 @@ public function getBody()
 	 */
 	public function geocodeAddress($address)
 	{
-		$url = 'http://maps.googleapis.com/maps/api/geocode/json?sensor=false&address=' . urlencode($address);
-		$response = $this->http->get($url);
+		$uri = JUri::getInstance('https://maps.googleapis.com/maps/api/geocode/json');
```

Comment:
This needs to be `new Uri('https://maps.googleapis.com/maps/api/geocode/json');`

The JUri class doesn't exist at the Framework level